### ISSUE TITLE
Error generating reference from .NET 4.0 Soap Service

### ISSUE THREAD

**Issue by law1185** [dotnet/wcf#1936]
Hi, I'm receiving this error when trying to create a Service Reference in VS 2017 (.NET Core 1.1) from a .NET 4.0 asmx service. I can generate both a Web Reference and Service reference in older versions of VS and .NET. I have tried deselecting the Reuse types checkbox and received the same result.

Scaffolding Code ...
Error:An error occurred in the tool. An item with the same key has already been added. Key: xxx

Failed to generate service reference.

**Comment by hongdai** [comment]
@law1185 Sorry to hear you are running to a issue!

Could you zip the log under %temp%\WCFConnectedService\<DateTime> and send it to me? It may give a clue what's going wrong.

**Comment by fabiano** [comment]
@hongdai,

I'm receiving the same message here when I try to add this service:

https://sandbox.api.giact.com/verificationservices/v5/InquiriesWS-5-8.asmx.

[2017_May_29_19_02_15.zip](https://github.com/dotnet/wcf/files/1036808/2017_May_29_19_02_15.zip)

**Comment by law1185** [comment]
@hongdai 

I have sent the zipped logs to you via email. Please let me know if you do not receive them.

Thanks!

**Comment by hongdai** [comment]
@law1185 I have examined both logs and did not find much clue.

I have tried to use https://sandbox.api.giact.com/verificationservices/v5/InquiriesWS-5-8.asmx to repro, it gave me an error "Client is not an authorized IP Address". Looks like the web site can only be accessed by authenticated users.

It will be greatly helpful if I can get a local repro. Is it possible for you to give me a repro service Uri without user authentication that everybody can access?

Thanks,
Hong
